tests/CocktailParty.tex

```diff
@@ -30,7 +30,7 @@ \subsection{Task}
 
 	\item \textbf{Sitting person:} At least one person is sitting and minding their own business (i.e. not looking directly to the robot). The robot must locate that person and ask if the person wants something to drink. The robot must also ask for the person's name and memorize them (i.e. execute a learning procedure of the name and the person's features). \\
 
-	\textbf{Remark:} At least one sitting person has an order to place. No sitting person will not attend robot's call.
+	\textbf{Remark:} At least one sitting person has an order to place. No sitting person will fail to attend to the robot's call.
```

Comment:
Indeed you resolved the double negation problem in a very logical way. However, the test prohibits sitting people to stand up, approach the robot, and place the order. To clarify even more, here is the snippet.

    when robot calls:
      if me.standing:
        me.aproach_to( robot.position )
        me.place_order()
      elsif me.sitting:
        me.ignore( robot )
      else:
        ...

If the word *attend* is still problematic we can change it for something like *sitting people will ignore robot's call*.